TimeMachineStatus/Localizable.xcstrings

```diff
@@ -114,19 +114,19 @@
         "de" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Neues Feedback senden..."
+            "value" : "Neues Feedback senden"
```

Comment:
Any reason you removed the `...` in many places? They are there intentionally to indicate a follow-up dialog or non-final action of some sort.

Reply:
In my opinion it is superfluous as user information, already it is visually visible. I was convinced I wrote it of the pull request description, my bad. 

Free to include them again.

Was the rest of the code correct? Missing translations are present